packages/web-react/transforms/__testfixtures__/transformAttributeExample.input.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { Button } from '../../src/components/Button';
```

Comment:
The import URL could be shorter, couldn't it?

Reply:
It could, but this will require setting an alias for that.

Reply:
https://www.typescriptlang.org/tsconfig#paths